central/complianceoperator/v2/checkresults/datastore/stats_views.go

```diff
@@ -43,3 +43,18 @@ type ResourceResultCountByProfile struct {
 	InconsistentCount  int    `db:"inconsistent_count"`
 	ProfileName        string `db:"compliance_profile_name"`
 }
+
+// ResourceResultsByProfile represents shape of the stats query for compliance operator results
+type ResourceResultsByProfile struct {
```

Comment:
As a user I might be interested in knowing when this result was created, so I know when the last scan was.
If UI wants to display it is on them. WDYTH?

Reply:
Looks like that is in the mocks, but that would not be this endpoint.

![Screenshot 2024-03-26 at 11 13 17 AM](https://github.com/stackrox/stackrox/assets/99685630/6f52edd5-366a-42b4-a6cb-c4ad52723c68)

Reply:
https://issues.redhat.com/browse/ROX-23356